backend/eslint.config.mjs

```diff
@@ -0,0 +1,19 @@
+import js from '@eslint/js';
```

Comment:
It actually cannot.  It may look like normal JS but that may be because babel has made it seem that way.  This file isn't transpiled into CJS, and so, it is a native module.  For NPM to support it, you either need to name it `mjs`, or declare that your whole project uses modules in your package.json.